python_src/src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, config_type: ConfigType, metadata_queue: Queue) -> None:
         self.archive_files: List[str] = []
 
     def convert(self) -> None:
-        max_tables_to_convert = 12
+        max_tables_to_convert = 6
```

Comment:
i dropped this so that each iteration happened faster. happy to revert if you prefer.